### DISCUSSION TITLE
「アプリケーションのエントリ」の質問

### DISCUSSION THREAD

**Discussion by tokuda109** [vuejs-jp/learn.nuxt.com#77] [ハンズオン Q&A] [answered]
https://github.com/vuejs-jp/learn.nuxt.com/blob/main/content/2.concepts/2.app-vue/index.md で何か困ったことや質問があれば遠慮なく質問して下さい

**Comment by kyagi-0826** [comment]
エントリがapp.vueとのことなのですが、これをindex.vueなどに変更はできたりするのでしょうか

**Reply by ubugeeei** [reply]
@wattanx 教えて！！

**Reply by wattanx** [reply]
~できた気がする~
気のせいでした

**Reply by atlansien** [reply]
~~vite.conig 経由?でできた気がします🤔~~
こちらも勘違いでした🙇

**Reply by wattanx** [reply, marked as answer]
公式では変更できるようなオプションを提供してなさそうに感じました
plugin とかを自分で作れば変更できるかもしれません。

↓で app.vue もしくは App.vue であることが固定になってるので
https://github.com/nuxt/nuxt/blob/abd5cae5410dcd80dc5a9e6d408eedf9f9cead4f/packages/nuxt/src/core/app.ts#L134-L140
